troi/tests/test_utils.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from troi.utils import discover_patches
+from troi.patch import Patch
+
+
+class TestPatches(unittest.TestCase):
+
+    def test_discover_patches(self):
+        patches = discover_patches()
+
+        assert len(patches) == 2
```

Comment:
are you expecting to have to change this every time we add a new built-in patch?

Reply:
Yes. My intention was to be a bit more explicit about adding new patches, but I don't feel strongly about it.